bindings_ffi/src/mls.rs

```diff
@@ -1333,6 +1399,40 @@ impl FfiGroup {
         Ok(group.is_active(group.mls_provider()?)?)
     }
 
+    pub fn consent_state(&self) -> Result<FfiConsentState, GenericError> {
+        let group = MlsGroup::new(
+            self.inner_client.context().clone(),
+            self.group_id.clone(),
+            self.created_at_ns,
+        );
+
+        let state = match group.consent_state()? {
+            ConsentState::Unknown => FfiConsentState::Unknown,
+            ConsentState::Allowed => FfiConsentState::Allowed,
+            ConsentState::Denied => FfiConsentState::Denied,
+        };
+
+        Ok(state)
+    }
+
+    pub async fn update_consent_state(&self, state: FfiConsentState) -> Result<(), GenericError> {
+        let group = MlsGroup::new(
+            self.inner_client.context().clone(),
+            self.group_id.clone(),
+            self.created_at_ns,
+        );
+
+        let consent_state = match state {
```

Comment:
Can make this into a separate implementation,

```

impl From<FfiConsentState> for ConsentState {
fn from(state: FfiConsentState) -> ConsentState {
        FfiConsentState::Unknown => ConsentState::Unknown,
            FfiConsentState::Allowed => ConsentState::Allowed,
            FfiConsentState::Denied => ConsentState::Denied,
}
}
```

Then wherever you're doing this match, can do `ConsentState::from(FfiConsentState)` or use the `into()` method, `let state: ConsentState = state.into()


doing this for the other enums as well should clean the code up a bit